budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/service/notification/Notification.java

```diff
@@ -3,5 +3,17 @@
 import java.io.Serializable;
 
 public abstract class Notification implements Serializable {
+    private NotificationType notificationType = NotificationType.warning;
 
+    public NotificationType getNotificationType() {
+        return notificationType;
+    }
+
+    public void setNotificationType(NotificationType notificationType) {
+        this.notificationType = notificationType;
+    }
+
```

Comment:
If you use the annotations `@Data` or `@Getter` and `@Setter` you don't need to write getters and setters by yourself.